src/components/block/DonationProjectCard/DonationProjectCard.module.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  height: 240px;
+  position: relative;
+}
+
+.bottombox {
+  bottom: 0;
+  display: flex;
+  flex-direction: row;
+  height: 72px;
+  position: absolute;
+  width: inherit;
```

Comment:
I'm going to respond to the comment thread from https://github.com/ShelterTechSF/sheltertech.org/pull/64#discussion_r502761928 here, since I think this line is what that ended up turning into in the latest version of this file.

> 👍 Width deleted. The position changed to inherent (would that work?).

I think you meant to say "_width_ changed to _inherit_"? In any case, I would not recommend doing this. Block-level elements by default take up the width of their containing parent, so it's usually unnecessary to ever explicitly set any property to `inherit` unless if you are overriding something back to the original behavior, such as in a media query.

However, now that we've cut down on the number of extraneous properties in this component, I can more easily see another issue with how we're doing this. When you [absolutely position an element](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout/In_Flow_and_Out_of_Flow#Absolute_Positioning), it takes it out of the normal flow of the document, which means it will not determine its width based on the width of its parent. Thinking about this more, since this component should always have a width equal to its parent's width, we probably actually want to make the parent a flex parent and set the `align-items` property so that the this bottom box always ends up aligned to the bottom.

> I read about the cascade and looked at examples but still a little iffy about it.

Without going into the technical minutia, the easiest way of thinking about this is that there are certain properties that will affect all children and descendants of the element that it is being applied to. For example, if I set the `font-family` of an element, it and all of its descendants will receive that font-family. There's no one simple rule for knowing which properties behave this way, so you'll always want to look this up for each property, but a general rule of thumb is that it applies for anything that would "make sense" to apply to all elements within a block.

This [StackOverflow post](https://stackoverflow.com/questions/5612302/which-css-properties-are-inherited) has a few lists of properties that generally behave this way.

Reply:
👍 Makes sense! Deleted width property with inherent value and added flexbox in parent while the bottom box uses align-self to position itself at the bottom it.